backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -313,17 +305,12 @@ open class SubmissionDatabaseService(
         submittedProcessedData: SubmittedProcessedData,
         organism: Organism,
         pipelineVersion: Long,
-    ): PreprocessingStatus {
+    ) {
```

Comment:
Function should be renamed to remove the "WithStatus" as there's no longer any status emitted

Reply:
https://github.com/loculus-project/loculus/pull/3125/commits/a72c1586cf148a28688cd4b3597a50377cf2a667